components/action/HistoriqueAction.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import InfoAction from 'components/action/InfoAction'
+import { Action } from 'interfaces/action'
+import { formatDayDate } from 'utils/date'
+
+interface HistoriqueActionProps {
+  action: Action
+}
+
+export function HistoriqueAction({ action }: HistoriqueActionProps) {
+  return (
+    <div className='border border-solid border-grey_100 rounded-medium pl-4 pr-4 mb-6'>
+      <h3 className='text-m-regular text-content_color mb-6 mt-4'>
```

Comment:
```suggestion
      <h2 className='text-m-bold text-content_color mb-4'>
```
à mettre au même niveau que Statut, le margin-bottom sera géré par le padding de la boîte englobant le tout